### PR TITLE
Quest and Questitem extensions

### PR DESCRIPTION
**Pull request by cljnilsson**
Nuff said.

### CODE REVIEW

File: wurst/_handles/QuestItem.wurst [hunk 1]
```diff
@@ -0,0 +1,24 @@
+package QuestItem
+
+public function questitem.description(string text)
```
Comment (Frotty) [inline wurst/_handles/QuestItem.wurst]: Shouldn't this be "setDescription" like the other ones?

File: wurst/_handles/Quest.wurst [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+    return IsQuestCompleted(this)
+
+public function quest.flash()
+    FlashQuestDialogButton()
+
+public function quest.remove()
+    DestroyQuest(this)
+
+
+    public enum QuestState
```
Comment (Frotty) [inline wurst/_handles/Quest.wurst]: "public enum QuestState" should not be indented, this is causing the travis build to fail

File: wurst/_handles/Quest.wurst [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+public function quest.discovered(boolean b)
+    QuestSetDiscovered(this, b)
+
+public function quest.isEnabled() returns boolean
+    return IsQuestEnabled(this)
+
+public function quest.isCompleted() returns boolean
+    return IsQuestCompleted(this)
+
+public function quest.flash()
```
Comment (Frotty) [inline wurst/_handles/Quest.wurst]: This shouldn't be an extension function, or?

**Comment by cljnilsson** [comment]
I think it should be an extension unless I am missing something entirely. Everything else is fixed.

**Comment by Frotty** [comment]
FlashQuestDialogButton doesn't take a quest handle though. And hence it doesn't change whatever quest you call it upon.

File: wurst/_handles/QuestItem.wurst [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+    construct(quest q, string text)
+        parent = q
+        qi = QuestCreateItem(parent)
+        qi.setDescription(text)
+
+    function isCompleted() returns boolean
+        return qi.isCompleted()
+    
+    function complete(boolean b)
+        qi.complete(b)
```
Comment (Cokemonkey11) [inline wurst/_handles/QuestItem.wurst]: missing eof lf

**Comment by cljnilsson** [comment]
Ohhh. Good point.

**Comment by Frotty** [comment]
so will you push a fix?? 😆

**Comment by Frotty** [comment]
I guess not. Well thanks anyway 👍

**Pull request merged**